e2e-tests/commands/insurance/assert-change-answers-page-url.js

```diff
@@ -9,14 +9,16 @@ import { INSURANCE_ROOT } from '../../constants/routes/insurance';
  * @param {String} Field ID for the field that is being changed/navigated to
  * @param {String} Optional fragment suffix for the URL. E.g 'heading', 'label'. Defaults to label.
  */
-const assertChangeAnswersPageUrl = (referenceNumber, route, fieldId, fragmentSuffix) => {
+const assertChangeAnswersPageUrl = (referenceNumber, route, fieldId, fragmentSuffix, noSuffix = false) => {
   let expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${route}`;
 
   if (fieldId) {
     expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${route}#${fieldId}`;
 
     if (fragmentSuffix) {
       expected = `${expected}-${fragmentSuffix}`;
+    } else if (noSuffix) {
+      expected = `${expected}`;
```

Comment:
Why is this needed? Feels strange, when changing an answer it should have a suffix so it goes to the label/heading of the form input

Reply:
the company details page has a heading.  It should focus on the heading which has data-cy of `heading` so it shouldn't be heading-label which is what happens otherwise

Reply:
Sounds like it can be achieved by passing `heading` to `fragmentSuffix` instead of adding a new param and condition here?

Reply:
that will make it `heading-heading` and if I leave it out it will be `heading-label`